bin/emqx

```diff
@@ -4,6 +4,19 @@
 
 set -e
 
+maybe_enable_debug() {
+    if [ "${BOOTSTRAP_DEBUG:-0}" = '1' ]; then
```

Comment:
```suggestion
    if [ ! -z $BOOTSTRAP_DEBUG ]; then
```

Reply:
It's quite confusing to use 'set anything' to enable. event when it's `false` or `FALSE` etc.